src/components/menu.svelte

```diff
@@ -1,250 +1,129 @@
-    text={'Open resume'}
-    click={() => {
-      const fileUpload = document.getElementById('file-upload');
-      if (fileUpload != null) fileUpload.click();
-      return null;
-    }}
-  />
-
-  <AddEntryButton
-    id={'saveResume'}
-    text={'Save resume'}
-    click={() => {
-      saveDataToJSONFile();
-      return null;
-    }}
-  />
-
-  <AddEntryButton
-    id={'printResume'}
-    text={'Print resume'}
-    click={() => {
-      window.print();
-      return null;
-    }}
-  />
-
-  <AddEntryButton
-    id={'clearResume'}
-    text={'[DEBUG] Clear resume data'}
-    click={() => {
-      localStorage.removeItem('saveData');
-      clearResumeStores();
-      return null;
-    }}
-  />
-
-  <input type="file" id="file-upload" accept=".json" style="display: none;" on:change={loadFile} />
-
-  <h2>Basic Information</h2>
-
-  <Input id={'basics_name'} label={'Name'} value={name} />
-  <Input id={'basics_title'} label={'Title'} value={label} />
-  <Input id={'basics_phone'} label={'Phone'} value={phone} />
-  <Input id={'basics_email'} label={'Email'} value={email} />
-
-  <h2>Work Experience</h2>
-  <div class="menu-content">
-    {#each $work as w, i}
-      <h3 class="submenu-header">
-        Work {i + 1}
-        <div class="label-controls">
-          {#if i > 0}
-            <button id="work_{i}_up" on:click={() => moveWork(i, true)}>Up</button>
-          {/if}
-          {#if i < $work.length - 1}
-            <button id="work_{i}_down" on:click={() => moveWork(i, false)}>Down</button>
-          {/if}
-          <button id="work_{i}_hide" on:click={() => hideWork(i)}>
-            {#if get(w.visible)}Hide{:else}Show{/if}
-          </button>
-          <button id="work_{i}_delete" on:click={() => deleteWork(i, get(w.name))}>Delete</button>
-        </div>
-      </h3>
-      <WorkMenu
-        {i}
-        visible={w.visible}
-        name={w.name}
-        position={w.position}
-        startDate={w.startDate}
-        endDate={w.endDate}
-        highlights={w.highlights}
-      />
-    {/each}
-    <AddEntryButton
-      id={'newWork'}
-      text={'Add new work entry'}
-      click={() => {
-        work.update((w) => {
-          w.push(new WorkStore());
-          return w;
-        });
-        return null;
-      }}
-    />
+<div id="menu-component">
+  <button id="open-button" class="open-button" on:click={() => (open = true)}>Open</button>
+  <div id="menu" class="menu {open ? 'open' : ''}">
+    <header id="menu-header">
+      {#if menuStackLength > 1}
+        <button id="back-button" on:click={() => (menuStack = pop(menuStack))}>Back</button>
+      {:else}
+        <button id="back-button" on:click={() => (open = false)}>Close</button>
+      {/if}
+      <h2 class="menu-title">MegaResume</h2>
+    </header>
+    <div class="menu-contents-container">
+      <!-- Menu contents components go in here -->
+      <MenuContents id="menu-contents-0" visible={visibleMenu === 'menu-contents-0'}>
+        <AddEntryButton
```

Comment:
These `AddEntryButtons` can just be button elements with the same props assigned to the button attributes. No need for another component.